.gitignore

```diff
@@ -5,5 +5,5 @@ __pycache__
 # Locally generated
 /settings
 .vscode/settings.json
-.DS_Store
-.bak
+*.DS_Store
```

Comment:
This ignore works for me. Can you give an example of where (platform, filename) where this helps?

Reply:
I'm a Windows user and recognized that as file type used on Mac systems. For some reason, I forgot that that was a file name and not a file extension, so I thought an asterisk was needed beforehand to make it match. I'll go ahead and change that back. As for adding ignores for temporary files created by vim and emacs, I'm not super familiar with the temporary files those programs create. The only files I'm recognizing within those example `.gitignore`s are swap files, so I don't have the knowledge to know which rules would be useful to add.

Reply:
OK, go ahead and fix that and I'll put the vim/emacs stuff in as a separate PR. Thanks!